articles/cms/wordpress/configuration.md

```diff
@@ -6,22 +6,35 @@ To configure the *Auth0 for WordPress* plugin, you will need to copy the *Domain
 
 You must first create an app in the Auth0 dashboard before you can configure the *Auth0 for WordPress* plugin. If you already have created the app you want to connect to WordPress, you can skip to the next section.
 
-1. Login to the [Auth0 dashboard](${manage_url}). (If you don't already have an Auth0 account, you can [create one](https://auth0.com).
-2. Navigate to the *Apps / APIs* page and click **+ New App / API**.
-3. In the *Create Application* window, name your app and click **Save**.
+1. Log in to the [Auth0 dashboard](http://${manage_url}). (If you don't already have an Auth0 account, you can [create one](https://auth0.com).
```

Comment:
Hey @khorne3 I think the `http://` should be removed from `(http://${manage_url})`. In the heroku preview at least it is displayed as `http//https://manage.auth0.com`.